docs/general/polkadot-v2.md

```diff
@@ -0,0 +1,234 @@
+  called [parachains](../learn/learn-parachains.md). Currently, one core is reserved for one chain
+  through [slot auction mechanism](../learn/learn-auction.md), although one core can be reserved
+  on-demand to multiple chains at different periods (see
+  [parathreads](../learn/learn-parathreads.md)). At the time of writing (mid 2023), on Polkadot,
+  there are around 50 cores independently operating in parallel.
+
+## Contracts on Cores
+
+If we treat Polkadot as a global supercomputer, cores can be used to run applications that can be
+deployed using parachains and core smart contracts, avoiding the need for custom chain
+infrastructure.
+
+From now _application_ will be used as a general term to describe anything that can use a Polkadot
+core to access secure and decentralized computation.
+
+## Core Usage in Polkadot 1.0
+
+In Polkadot 1.0, applications produced blocks at a fixed rate of 12 seconds, whether needed or not.
+This led to inefficient energy allocation and economic incentives for producing full blocks under
+heavy traffic and empty blocks under light traffic.
+
+The figure below shows the core usage for Polkadot 1.0, where the horizontal axis is time, and each
+row represents a core. Colors show different parachains, each using one core (i.e., one parachain,
+one core formula).
+
+![core-usage-dumb](../assets/core-usage-dumb.png)
+
+The above setup allowed a **simple and secure, sharded execution environment**.
+
+However, to achieve full efficiency blocks must be produced when needed, and the system must target
+full block capacity lowering the probability of incentivizing validators to build blocks half full
+or, worse, empty.
+
+## From Slot Auctions to Coretime Marketplace
+
+The end product of blockchains is [**Blockspace**](./polkadot-v1.md#polkadots-blockspace).
+Applications need to access Polkadot's blockspace, and the entry points to blockspace are the cores.
+Thus, applications will need to reserve some time on cores or **Coretime** to gain the right to
+access Polkadot's secure blockspace and interoperability for a finite period.
+
+Cores must be agile and general: they can change what job they run as easily as a modern CPU. It
+follows that the procurement of those cores must be agile as well.
+
+The slot auction mechanism is not agile, creates high entry barriers, and is designed for
+long-running single applications (i.e., the original Polkadot vision proposed in the whitepaper).
+
+We depart from the classic lease auctions and propose an agile marketplace for coretime, where
+essentially **coretime becomes a commodity that can be tokenized, sold and traded**. This setup
+maximizes the agility of Polkadot and lets the market figure out the best solution needed for
+applications to be successful.
+
+Revenues from coretime sales can be burnt, used to fund the Treasury, split between coretime
+providers (i.e., validators), or used for a mix of those options. The topic is currently under
```

Comment:
IIRC, the last method (splitting between coretime providers) is no longer considered. Check the latest discussion on RFC 10 for more details.